### ISSUE TITLE
Question: How to merge pull requests to your own fork

### ISSUE THREAD

**Issue by 10010101001** [endless-sky/endless-sky#4114]
For example, if I wanted to merge #3583 into my fork, how do I do it?

**Comment by MCOfficer** [comment]
Ok, so first off: this doesn't belong here. It's a basic git question, has little to do with ES, and even if it had it wouldn't be worth filing an issue. It should be closed.

---

Be aware that once the PR to your fork is merged, any subsequent changes to tehh's branch will not be reflected on your fork. If you want to test a feature, it's generally a better idea to just clone & compile tehh's fork.

---

i'll use #3583 as example, but the same procedure applies for all other PRs.

**1. Find the PR's head fork:**

![image](https://user-images.githubusercontent.com/22377202/50742003-98d8eb00-1205-11e9-9878-926383942079.png)
you're looking for `tehhowch:ArfectaUpdate`. This means that the PR comes from the branch `ArfectaUpdate` on `tehhowch`'s fork.
It's a shame Github doesn't provide a link to this branch, so you have to go to tehhowch's profile, select his fork, and switch to the `ArfectaUpdate` branch:
![image](https://user-images.githubusercontent.com/22377202/50742022-ee14fc80-1205-11e9-8c4f-64d908966e79.png)

**2. Open a new PR:**
The cool thing about Git (and/or Github) is that everyone can open PRs, even from branches that are not under their control. That means that you can open a PR from tehhowch's branch to one of yours.

![image](https://user-images.githubusercontent.com/22377202/50742038-40eeb400-1206-11e9-9f78-d5a0f1688fdd.png)

Click "Pull Request". You will be taken to a page that lists all changes and allows you to change where to PR from - and where to.



Before doing anything in a rush, double check these things:
a) did you select **your own fork** as base fork?
b) Is your master up-to-date with the current ES master? (This can lead to ugly merge conflicts)

Are you sure about those points? Ok, then click "Create Pull Request". Voila, you now have a PR to your own branch.
